README.md

```diff
@@ -8,7 +23,8 @@ Note - if you are pulling, look at the [branches section](#branches)
 Firefly builds are available on Docker and the notes for using it are on the [Docker Page](https://hub.docker.com/r/ipac/firefly).
```

Comment:
Perhaps move this sentence up before the "Note" one, to put it closer to the above mention of Docker.  Say "... and additional notes for using it are ..." to tie it together more.